config/redirects.js

```diff
@@ -2119,8 +2119,9 @@ const redirects = [
       '/protocols/saml/saml-configuration/auth0-as-identity-provider',
       '/protocols/saml-protocol/configure-auth0-as-saml-identity-provider',
       '/configure/saml-configuration/configure-auth0-saml-identity-provider',
+      /authenticate/protocols/saml/saml-sso-integrations/configure-auth0-saml-identity-provider,
```

Comment:
missing single quotes
```suggestion
      `/authenticate/protocols/saml/saml-sso-integrations/configure-auth0-saml-identity-provider`,
```

Reply:
I thought the semgrep test checked for syntax?